### PR TITLE
Bug: Toggle button in Adapter Selection.

### LINKED ISSUE #640: Toggle button in Adapter Selection does not change state on deny
**Describe the bug**
On trying to enable bluetooth from the app the toggle button does not change it state to off if we click on deny on anywhere on screen.

**To Reproduce**
1. Click on OBD selection in Dasboard fragment.
2. Click on toggle button.
3. Click on deny or anywhere on screen.
4. Toggle button switches to ON state rather it should be in off state if deny or nothing is selected.

**Expected behavior**

Reset toggle button to off state if no selection is made or deny is clicked. Also we can add ON /OFF instead of simple toggle button.

### PR DESCRIPTION
**Pull request by asaikarthikeya**
Fixes #640 

After -

https://user-images.githubusercontent.com/70392921/115015310-1275de00-9ed1-11eb-926c-de3e9c269a21.mp4

### CODE REVIEW

**Comment by Akash-Ramjyothi** [comment]
LGTM.

**Review by Akash-Ramjyothi — APPROVED** [review]

**Comment by asaikarthikeya** [comment]
@SebaDro Please review. Thank you.

File: org.envirocar.app/src/org/envirocar/app/views/obdselection/OBDSelectionActivity.java [hunk 1]
```diff
@@ -103,12 +103,12 @@ private void setupBluetoothSwitch() {
...
-        mSwitch.setChecked(mBluetoothHandler.isBluetoothEnabled());
 
         mSwitch.setOnCheckedChangeListener((buttonView, isChecked) -> {
             if (isChecked) {
-                //                mSwitch.setChecked(false);
+                //  If Bluetooth is allowed then mSwitch checked
                 mBluetoothHandler.enableBluetooth(OBDSelectionActivity.this);
+                mSwitch.setChecked(mBluetoothHandler.isBluetoothEnabled());
             } else {
                 mBluetoothHandler.disableBluetooth(OBDSelectionActivity.this);
```
Comment (SebaDro) [inline org.envirocar.app/src/org/envirocar/app/views/obdselection/OBDSelectionActivity.java:113]: You should also consider the other way: if you decline deactivating Bluetooth, the toggle button also changes its stage, which should not happen.

**Review by SebaDro — CHANGES_REQUESTED** [review]

**Review by SebaDro — APPROVED** [review]

File: org.envirocar.app/src/org/envirocar/app/views/obdselection/OBDSelectionActivity.java (hunk 1 above)
Comment (asaikarthikeya) [reply org.envirocar.app/src/org/envirocar/app/views/obdselection/OBDSelectionActivity.java:113]: I see no request when we turn off the Bluetooth. Updated accordingly `mSwitch.setChecked(false)`

**Pull request merged**